src/views/Chat/Chat.jsx

```diff
@@ -24,12 +24,11 @@ export default function Chat() {
   };
 
   useEffect(() => {
-    getMessages().then(setMessages);
-    // TODO: Subscribe to message changes, using handleMessageReceived
-    // as the callback function
-    // Don't forget to return a cleanup method (`unsubscribe`)
-    // from this useEffect!
-  }, []);
+    getMessages()
+      .then(setMessages)
+      .then(subscribe)
+      .finally(() => unsubscribe);
```

Comment:
Super close here Brenden. We will actually want to call subscribe with a parameter of our callback function (handleMessageReceived)